samples/crypto/spake2p/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
```

Comment:
Suggest removing that `#` to mark a clear distinction between header and start of configuration.

Reply:
If I fix this we should do it for all crypto samples to have the config file structure aligned.
I don't want to do that in this bugfix commit.